.changeset/bright-gorillas-joke.md

```diff
@@ -0,0 +1,12 @@
+---
+'@finos/legend-extension-dsl-data-space': minor
+---
+
+Added a new query creation mode from data space, this will mode can be accessed from `Legend Query` using the `url pattern`
+
+```
+/query/extensions/{groupId}:{artifactId}:{versionId}/{dataSpacePath}/{executionContext}/{runtimePath}?
```

Comment:
why is runtimePath an option here

Reply:
Since people are picking the `execution context` of a data space which already specified with a `default runtime`. When people pick another option other than the `default runtime`, we will put that in the url with this param